commons/AndroidManifest.xml

```diff
@@ -1,7 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="fr.free.nrw.commons"
-    android:versionCode="18"
-    android:versionName="1.3" >
+    android:versionCode="19"
+    android:versionName="1.31" >
```

Comment:
How about bumping to 2.0 after nearby categories and fuzzy search are
implemented? :-)

On Mon, Nov 9, 2015 at 2:08 PM, Josephine Lim <notifications@github.com>
wrote:

> In commons/AndroidManifest.xml
> <https://github.com/nicolas-raoul/apps-android-commons/pull/29#discussion_r44241689>
> :
>
> > @@ -1,7 +1,7 @@
> >  <manifest xmlns:android="http://schemas.android.com/apk/res/android"
> >      package="fr.free.nrw.commons"
> > -    android:versionCode="18"
> > -    android:versionName="1.3" >
> > +    android:versionCode="19"
> > +    android:versionName="1.31" >
>
> Oh, okay! I had no idea, I thought it worked like decimals. So I'll change
> it to 1.4, then in the future if we release a version with a big change
> like category suggestions we can skip to 2.0?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nicolas-raoul/apps-android-commons/pull/29/files#r44241689>
> .
>